deployment-platforms/vercel-graphql/api/_lib/context.ts

```diff
@@ -1,6 +1,8 @@
 import { PrismaClient } from '@prisma/client'
 
-const prisma = new PrismaClient()
+const prisma = new PrismaClient({
+  log: ['query', 'info', 'warn'],
```

Comment:
Do we want to keep this logging in the examples?

Reply:
Yes.

Reply:
My thought was that this provides useful insight into what's happening with the application as queries come in.

Reply:
I would put it commented with a comment "Useful for debugging..." or remove it personally